packages/app/pages/Api/orgType.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * This exports a React element which displays a list of all Organization Types
+ *
+ * @package: REA app
+ * @author:  Ryan Guinn <guinnrd@msoe.edu>
+ * @since:   2018-02-02
+ */
+
+import * as React from "react";
+import "./api.css"
+import GetOrganizationTypes from "../../../ui-bindings/OrganizationType/getAllOrganizationTypes.js";
+
+export const OrganizationType = (props) => {
+  return(
+    <div>
+      <div>id: {props.id}</div>
+      <div>name: {props.name}</div>
+      <br/>
+    </div>
+  );
+};
+
+export const OrganizationTypeList = GetOrganizationTypes(({ orgTypeList, loading, error}) => {
+  return (
+    loading ? <strong>Loading...</strong> : (
+      error ? <p style={{color: "#F00"}}>API error</p> : (
+        <div>
+          {orgTypeList.map( (orgType) => (<OrganizationType key={orgType.id} id={orgType.id} name={orgType.name}/>))}
```

Comment:
You should pass the whole orgType to the react element, not the individual properties. It dosen't matter for this PBI, but it is better practice. if you had a lot of queries and a lot of properties, it would be a mess. See what I did in the agent page.

Reply:
I agree with Steven that this would be good practice to start incorporating, otherwise everything else is technically fine.